docs/resources/project.md

```diff
@@ -84,7 +84,7 @@ resource "project" "myproject" {
 - **id** (String) The ID of this resource.
 - **max_storage_in_gibibytes** (Number) Storage quota in GiB. Must be 1 or larger. Set to -1 for unlimited storage. This is translated to binary bytes for Artifactory API. So for 1TB quota, this should be set to 1024 (vs 1000) which will translate to 1099511627776 bytes for the API.
 - **member** (Block Set) Member of the project. Element has one to one mapping with the [JFrog Project Users API](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-UpdateUserinProject). (see [below for nested schema](#nestedblock--member))
-- **repos** (Set of String) List of existing repo keys to be assigned to the project.
+- **repos** (Set of String) List of existing repo keys to be assigned to the project. You can set a maximum 100 repo keys. (se [below for limitations on the maximum number of repos](#repoNumLimitations)). You can override the default value by setting an environment value `REPO_LIMIT_OVERRIDE` to `true`. The default value of `REPO_LIMIT_OVERRIDE` is `false` when not set. E.g. To override the default maximum allowable elements of `repos` attribute use `export REPO_LIMIT_OVERRIDE=true`.
```

Comment:
You have a typo here

`se[`

Reply:
And you're using square brackets. What formatting are you trying to achieve

Reply:
The description does not explain clearly how that env var affects the limitation of 100 repo keys.

Reply:
updated